doc/nrf/releases/release-notes-latest.rst

```diff
@@ -24,7 +24,34 @@ The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repositor
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-*
+* Bluetooth
+
+  * Support for LE Advertising Extensions has been added.
+  * New application-controlled data length and PHY update APIs.
+  * Legacy OOB pairing support has been added.
+  * Multiple improvements to OOB data access and pairing.
+  * Multiple bug fixes and improvements
+  * deprecated BT_LE_SCAN_FILTER_DUPLICATE, use BT_LE_SCAN_OPT_FILTER_DUPLICATE instead
+  * deprecated BT_LE_SCAN_FILTER_WHITELIST, use BT_LE_SCAN_OPT_FILTER_WHITELIST instead
+  * deprecated bt_le_scan_param::filter_dup, use bt_le_scan_param::options instead
+  * deprecated bt_conn_create_le(), use bt_conn_le_create() instead
+  * deprecated bt_conn_create_auto_le(), use bt_conn_le_create_auto() instead
+  * deprecated bt_conn_create_slave_le(), use bt_le_adv_start() instead with
+    bt_le_adv_param::peer set to the remote peers address.
+  * deprecated BT_LE_ADV_* macros, use BT_GAP_ADV_* enums instead
+
+* Bluetooth LE Controller:
+
+  * The Controller is now 5.2 compliant.
+  * PHY support is now configurable.
+  * Only control procedures supported by the peer are now used.
+  * The nRF52820 IC is now supported
+  * The legacy Controller has been removed.
+
+* Bluetooth Mesh
+
+  * The net_idx parameter has been removed from the Health Client model
+    APIs since it can be derived (by the stack) from the app_idx parameter
```

Comment:
```suggestion
  * Removed the ``net_idx`` parameter from the Health Client model APIs because it can be derived (by the stack) from the ``app_idx`` parameter.
```